shell/src/main/java/alluxio/cli/fs/command/PinCommand.java

```diff
@@ -67,7 +67,7 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
     if (!invalidMediumType.isEmpty()) {
       throw new IllegalArgumentException("Invalid medium to pin the file. "
           + String.join(",", invalidMediumType) + " are invalid. "
-          + String.join(",", availableMediumList) + " are valid medium types");
+          + String.join(",", validMediumType) + " are valid medium types");
```

Comment:
IDE is warning about `validMediumType` is modified but never queried. This should be a typo.